cmd/aws-sigv4-proxy/main.go

```diff
@@ -42,6 +42,7 @@ var (
 	strip                  = kingpin.Flag("strip", "Headers to strip from incoming request").Short('s').Strings()
 	roleArn                = kingpin.Flag("role-arn", "Amazon Resource Name (ARN) of the role to assume").String()
 	signingNameOverride    = kingpin.Flag("name", "AWS Service to sign for").String()
+	signingHostOverride    = kingpin.Flag("sign-host", "Host to sign for").String()
```

Comment:
FYI - it is possible to replace (hide) this feature via env variable (i.e. `SIGNING_HOST` ) to avoid multiplying parameters if needed.

Similar to idea of [aws-sigv4-proxy/pull/2][1]

---
[1]: https://github.com/awslabs/aws-sigv4-proxy/pull/2